packages/features/eventtypes/components/tabs/assignment/EventTeamAssignmentTab.tsx

```
@@ -719,8 +869,16 @@
                       {!!(
                         eventType.team?.rrTimestampBasis &&
                         eventType.team?.rrTimestampBasis !== RRTimestampBasis.CREATED_AT
-                      ) ? (
-                        <Tooltip content={t("rr_load_balancing_disabled")}>
+                      ) || hostGroups?.length > 1 ? (
+                        <Tooltip
+                          content={
+                            !!(
+                              eventType.team?.rrTimestampBasis &&
+                              eventType.team?.rrTimestampBasis !== RRTimestampBasis.CREATED_AT
+                            )
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)